README.md

```diff
@@ -19,15 +20,51 @@
 
 ---
 
-Wn is a Python library for exploring information in wordnets. Install
-it from PyPI and download some data:
+Wn is a Python library for exploring information in wordnets. 
+
+## Installation
+
+Install it from PyPI using **pip** and download some data:
 
 ```console
 $ pip install wn
 $ python -m wn download oewn:2022  # the Open English WordNet 2022
 ```
 
-Then start exploring:
+Or, install using **conda**, from conda-forge channel:
+
+```sh
+# Method-1
+conda install -c conda-forge wn
+
+# Method-2
+# Using: conda install CHANNELNAME::PACKAGENAME
+conda install conda-forge::wn
+```
+
+> 🔔 Tip: **Speeding up Conda Installations**
+>
+> To speedup conda installations, alternatively, consider installing `mamba` 
+> from conda-forge channel and then replace all `conda` commands with `mamba` 
+> as a drop-in replacement.
+>
+> ```sh
+> conda install -c conda-forge mamba
+> mamba install -c conda-forge wn
+> ```
+
+For more helpful tips on conda, refer to this handy [Conda Cheatsheet ↗️][#conda-cheatsheet]. :fire:
+
+> ℹ️ The conda-forge recipe for this python library is maintained at the 
+> following conda-forge **feedstock** repository.
+>
+> - **Feedstock**: <https://github.com/conda-forge/wn-feedstock>
+
+[#conda-cheatsheet]: https://conda.io/projects/conda/en/latest/_downloads/a35958a2a7fa1e927e7dfb61ebcd69a9/conda-4.14.pdf
+
+## Getting Started
+
+Once installed, then start exploring:
```

Comment:
The sentence is a bit weird. Maybe "Once you've installed the software and some data, you can start exploring:"

Reply:
Yes, it sounds weird. 

Here are my suggestions:

1. _Once installed, start exploring:_
2. _Now start exploring:_

Reply:
I like simple. How about "Now start exploring:". Thanks for the suggestion

Reply:
Done.